christfellowshipchurch/web-app-v2
# Updated icons

### About
Deleted duplicated icons and created an icons library that was easy to visualize. 
Replaced all icons from the Figma new icons library from FontAwesome

### Test Instructions
Open test link + /icons-rendering to see the new icons library
Here are the things you can test:
- [ ] The names of the icons match the names on Figma.
- [ ] Icons are organized and sizing is consistent except for the few that will be smaller (angles, arrows, plus and x signs).
- [ ] Figma is organized and well-documented.
- [ ] You can search for icons all over our website and see if they display properly where they are being used.

### Screenshots
![Screen Shot 2023-09-26 at 2 06 28 PM](https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/9eddca80-c7e8-4f06-b9c0-b1b591600783)


### Closes Tickets
N/A

### Related Tickets
[CFDP-2705](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2705)


[CFDP-2705]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2705?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CFDP-2751](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2751)

[CFDP-2751]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2751?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
